letsencrypt-auto-source/pieces/bootstrappers/rpm_python3.sh

```diff
@@ -0,0 +1,23 @@
+# If new packages are installed by BootstrapRpmPython3 below, this version
+# number must be increased.
+BOOTSTRAP_RPM_PYTHON3_VERSION=1
+
+BootstrapRpmPython3() {
+  # Tested with:
+  #   - CentOS 6
+
+  InitializeRPMCommonBase
+
+  # EPEL uses python34
+  if $TOOL list python34 >/dev/null 2>&1; then
+    python_pkgs="python34
+      python34-devel
+      python34-tools
```

Comment:
Did you determine that this package is necessary?

Reply:
nope, but I don't know what we were using it for in the first place, in the non-3 python version.

Reply:
Looking into this, I found #1823. There and in the linked blogpost, one person claims `python-tools` and `python-pip` need to be installed without really saying why. Initially, the post talks about CentOS 7 and [later](https://mangolassi.it/topic/7127/setting-up-letsencrypt-on-a-centos-7-nginx-proxy) says they are necessary to fix `InsecurePlatformWarning`. A few things:

1. The installation of these packages do not fix `InsecurePlatformWarning` per my testing.
2. `InsecurePlatformWarning` is not a fatal error.
3. The error isn't shown in normal output (unless you asked for increase verbosity) because the error is irrelevant as we're verifying the hashes of the downloaded packages.

With that said, I looked into the contents of the `python34-tools` package using `repoquery` as described [here](https://stackoverflow.com/questions/104055/how-to-list-the-contents-of-a-package-using-yum). This package contains:

1. [2to3](https://docs.python.org/2/library/2to3.html) and [IDLE](https://docs.python.org/2/library/idle.html) placed in the user's PATH.
2. A bunch of Python developer tools and demos that aren't placed in the user's PATH or in a location importable by Python.

I think this package is cruft and can be removed, especially in the Python 3.4 case which doesn't (currently) have `InsecurePlatformWarning` problems. I think this cleans up `certbot-auto` and reduces our dependency count a bit, but the installed package is only 1.2 MB. If you'd prefer to leave it in despite what I've found, that's fine with me.

Reply:
It sounds like this isn't a python3-specific problem, which is what this PR is about fixing. If we're removing it, it belongs in a separate PR that can be separately reverted if something goes wrong. Happy to also get that in for this release.